### ISSUE TITLE
Error or not error, that is the question

### ISSUE THREAD

**Issue by paramecie** [rgriebl/brickstore#319]
This very test file shows an error:

![image](https://user-images.githubusercontent.com/32078594/112368844-cf628980-8cdb-11eb-88c3-8baf8524c78f.png)

OK but if I set the error item "Excluded" it's more complicated.

First it shows an error, but isn't considered as an error (see top right):

![image](https://user-images.githubusercontent.com/32078594/112369042-0c2e8080-8cdc-11eb-9c42-185ca84effd8.png)

But in fact, it's still... an error.
If I try CTRL+E CTRL+U (mass upload), it says it's an error!

![image](https://user-images.githubusercontent.com/32078594/112369118-236d6e00-8cdc-11eb-8094-ecb51a4448e4.png)

Questions are multiple:
* if I upload this while the item being Excluded, why does it consider the item (it shouldn't), and shows an error?
* why does the price is in Red (an error) and not in error count because excluded?

I've found this bizarre behaviour somewhere else, but can't remember...
In short, can you check/reconsider what's shown as an error, what's counted as an error (should be the same?), which items are checked in a selection for upload or any other feature (in short: ignore excluded)?

**Comment by rgriebl** [comment]
Just checked all code paths: 
* All XML exports ignore excluded items.
* The function to calculate the error count has a flag `ignoreExcluded`.  It's called with that being set for the counter at the top and without for drawing the red boxes and the export check function

What would make the most sense for an excluded item? ignore error conditions or not?

**Comment by paramecie** [comment]
Yes, I guess:

* excluded items should NOT be checked when mass uploading, for example and in this above case - as they're excluded
* excluded items may show no error at all, as anyway they won't (shouldn't) be used in any case, as they're exluded.

BTW I just checked the excluded item is actually printed in the standard printing layout - I guess it shouldn't?

**Comment by rgriebl** [comment]
Regarding printing: the new printing code is what-you-see-is-what-you-get: whatever is on the screen will be exactly printed as is.

**Comment by paramecie** [comment]
OK Boss, you're the Boss :-) I'll try this when patch is released.

**Comment by paramecie** [comment]
Build 400 (yey!) - OK excluded don"t show error anymore. That seems to me less annoying.

I'm still puzzled by the fact it considers an excluded as an item (but later on it simply ignores it during the export):

![image](https://user-images.githubusercontent.com/32078594/112391678-91c02980-8cf8-11eb-8c0c-fccb25dc0ef6.png)

What would be (IMO) even better would be #266 ...

I'm not sure, just saying ;-)
